derive/src/lib.rs

```diff
@@ -80,11 +83,12 @@ pub fn eth_abi(args: TokenStream, input: TokenStream) -> TokenStream {
 					let client_name = args.get(1).expect("Failed to parse an client name argument");
 					let endpoint = generate_eth_endpoint(&endpoint_name, &intf);
 					let client = generate_eth_client(client_name, &intf);
-					let endpoint_use: syn::Ident = format!("self::pwasm_abi_impl::{}", &endpoint_name).into();
-					let client_use: syn::Ident = format!("self::pwasm_abi_impl::{}", &client_name).into();
+					let endpoint_use: syn::Ident = format!("self::{}::{}", &mod_name, &endpoint_name).into();
+					let client_use: syn::Ident = format!("self::{}::{}", &mod_name, &client_name).into();
 					let generated = quote! {
 						#intf
-						mod pwasm_abi_impl {
+						#[allow(non_snake_case)]
+						mod #mod_name_ident {
```

Comment:
The purpose of this is to give unique name for mod. Allows to declare multiple endpoints in one namespace